yapapi/log.py

```diff
@@ -441,7 +441,7 @@ def _handle(self, event: events.Event):
             provider_info = self.agreement_provider_info[event.agr_id]
             reason = str(exc) or repr(exc) or "unexpected error"
             self.logger.error(
-                "Failed to accept invoice from '%s', reason: %s", provider_info, reason
+                "Failed to accept Invoice or Debit Note from '%s', reason: %s", provider_info, reason
```

Comment:
This line is too long
```suggestion
                "Failed to accept an invoice or a debit note from '%s', reason: %s",
                provider_info,
                reason,
```